src/deluge/model/mod_controllable/mod_controllable_audio.cpp

```diff
@@ -2047,3 +2047,37 @@ char const* ModControllableAudio::getSidechainDisplayName() {
 		return l10n::get(STRING_FOR_FAST);
 	}
 }
+
+/// displays names of parameters assigned to gold knobs
+void ModControllableAudio::displayOtherModKnobSettings(uint8_t whichModButton, bool on) {
```

Comment:
Could you add a doc comment explaining what this is doing for oled and non-oled contexts?

Reply:
> Could you add a doc comment explaining what this is doing for oled and non-oled contexts?

Done!